litmus/libdir/armv8+sve-qemu.cfg

```
@@ -0,0 +1,13 @@
+size_of_test = 10k
+number_of_run = 100
+avail = 4
+limit = true
+memory = direct
+stride = 1
+carch = AArch64
+barrier = userfence
+smt = 4
+smt_mode = seq
+affinity = incr0
+ccopts = -O2 -march=armv8-a+sve
+crossrun = qemu:qemu-aarch64
```

It may be appropriate to specify the usual name for aarch64 cross compiler and static linking:
```suggestion
crossrun = qemu:qemu-aarch64
gcc = aarch64-linux-gnu-gcc
linkopt = -static
```
static linking is useful when the aarch64 `.so` for libc is not installed, which is often the case.